nepalingo-web/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import useDictionary, { DictionaryProps, DictionaryResponse } from '../../hooks/useDictionary';
+
+const SearchBar: React.FC = () => {
+  const [searchTerm, setSearchTerm] = useState<string>('');
+  const language: DictionaryProps['language'] = 'newari'; 
```

Comment:
can the language be taken in as a prop in line 6? That way whoever is calling SearchBar can determine the language used